### ISSUE TITLE
AWS cross-az failover not working

### ISSUE THREAD

**Issue by chen23** [F5Networks/f5-cloud-failover-extension#58]
## Do you already have an issue opened with F5 support?
No

## Description
When deploying the AWS CFT w/ CFE 1.6.1 the failover is not working properly.

The logs seem to indicate that is trying to associate with the wrong address.  Furthermore it appears to remove the secondary private address from the other BIG-IP ENI that makes it not possible to fail back.

```
Thu, 07 Jan 2021 20:44:09 GMT - fine: [f5-cloud-failover] Filtered Buckets: {"0":{"Bucket":"erchen-ipsec-s3bucket-1w873rmar30r2","TagSet":[{"
Key":"aws:cloudformation:stack-name","Value":"erchen-ipsec"},{"Key":"aws:cloudformation:logical-id","Value":"S3Bucket"},{"Key":"aws:cloudform
ation:stack-id","Value":"arn:aws:cloudformation:us-east-1:317124676658:stack/erchen-ipsec/09438960-5032-11eb-8b54-12be25e2835f"},{"Key":"f5_c
loud_failover_label","Value":"erchen-ipsec"}]}}
Thu, 07 Jan 2021 20:44:09 GMT - finest: [f5-cloud-failover] Cloud Provider initialization complete
Thu, 07 Jan 2021 20:44:09 GMT - finest: [f5-cloud-failover] Failover initialization complete
Thu, 07 Jan 2021 20:44:09 GMT - finest: [f5-cloud-failover] Downloading data from: f5cloudfailover/f5cloudfailoverstate.json
Thu, 07 Jan 2021 20:44:09 GMT - finest: [f5-cloud-failover] Download stateFile: {"taskState":"RUNNING","message":"Failover running","timestam
p":"2021-01-07T20:43:12.986Z","instance":"ip-10-1-3-191.ec2.internal","failoverOperations":{"addresses":{"publicAddresses":{"52.204.125.179":
{"current":{"PrivateIpAddress":"10.1.10.232","AssociationId":"eipassoc-0e7be69a7125016b2"},"target":{"PrivateIpAddress":"10.1.13.184","Networ
kInterfaceId":"eni-0c2e58830dd22446f"},"AllocationId":"eipalloc-08f88ca9723dd1518"}},"interfaces":{"disassociate":[{"networkInterfaceId":"eni
-0441941ac4b5dca76","addresses":[{"address":"10.1.10.232","publicAddress":"52.204.125.179"}]}],"associate":[{"networkInterfaceId":"eni-0c2e58
830dd22446f","addresses":[{"address":"10.1.10.232","publicAddress":"52.204.125.179"}]}]},"loadBalancerAddresses":{}},"routes":{"operations":[
]}}}
Thu, 07 Jan 2021 20:44:09 GMT - finest: [f5-cloud-failover] taskState: {"taskState":"RUNNING","message":"Failover running","timestamp":"2021-
01-07T20:43:12.986Z","instance":"ip-10-1-3-191.ec2.internal","failoverOperations":{"addresses":{"publicAddresses":{"52.204.125.179":{"current
":{"PrivateIpAddress":"10.1.10.232","AssociationId":"eipassoc-0e7be69a7125016b2"},"target":{"PrivateIpAddress":"10.1.13.184","NetworkInterfac
eId":"eni-0c2e58830dd22446f"},"AllocationId":"eipalloc-08f88ca9723dd1518"}},"interfaces":{"disassociate":[{"networkInterfaceId":"eni-0441941a
c4b5dca76","addresses":[{"address":"10.1.10.232","publicAddress":"52.204.125.179"}]}],"associate":[{"networkInterfaceId":"eni-0c2e58830dd2244
6f","addresses":[{"address":"10.1.10.232","publicAddress":"52.204.125.179"}]}]},"loadBalancerAddresses":{}},"routes":{"operations":[]}}}
Thu, 07 Jan 2021 20:44:09 GMT - finest: [f5-cloud-failover] Failover is already executing
Thu, 07 Jan 2021 20:44:09 GMT - finest: [f5-cloud-failover] Downloading data from: f5cloudfailover/f5cloudfailoverstate.json
Thu, 07 Jan 2021 20:44:10 GMT - finest: [f5-cloud-failover] Download stateFile: {"taskState":"RUNNING","message":"Failover running","timestam
p":"2021-01-07T20:43:12.986Z","instance":"ip-10-1-3-191.ec2.internal","failoverOperations":{"addresses":{"publicAddresses":{"52.204.125.179":
{"current":{"PrivateIpAddress":"10.1.10.232","AssociationId":"eipassoc-0e7be69a7125016b2"},"target":{"PrivateIpAddress":"10.1.13.184","Networ
kInterfaceId":"eni-0c2e58830dd22446f"},"AllocationId":"eipalloc-08f88ca9723dd1518"}},"interfaces":{"disassociate":[{"networkInterfaceId":"eni
-0441941ac4b5dca76","addresses":[{"address":"10.1.10.232","publicAddress":"52.204.125.179"}]}],"associate":[{"networkInterfaceId":"eni-0c2e58
830dd22446f","addresses":[{"address":"10.1.10.232","publicAddress":"52.204.125.179"}]}]},"loadBalancerAddresses":{}},"routes":{"operations":[]}}}
Thu, 07 Jan 2021 20:44:11 GMT - fine: [f5-cloud-failover] Disassociating address using eipassoc-0e7be69a7125016b2
Thu, 07 Jan 2021 20:44:11 GMT - finest: [f5-cloud-failover] Function error, retrying: The association ID 'eipassoc-0e7be69a7125016b2' does not exist Retries left: 39
Thu, 07 Jan 2021 20:44:12 GMT - fine: [f5-cloud-failover] Associating [{"address":"10.1.10.232","publicAddress":"52.204.125.179"}] to eni-0c2e58830dd22446f
Thu, 07 Jan 2021 20:44:13 GMT - finest: [f5-cloud-failover] Function error, retrying: Address does not fall within the subnet's address range Retries left: 39
```

## Environment information
For bugs, enter the following information:
- Cloud Failover Extension Version: 1.6.1
- BIG-IP version: 15.1.2 0.0.9
- Cloud provider: AWS

## Severity Level
For bugs, enter the bug severity level. Do not set any labels.

Severity: 2

**Comment by nmenant** [comment]
Eric, could you check if this is fixed with 1.7.0, i believe it should be the case. Thanks

**Comment by shyawnkarim** [comment]
Closing.

Please reopen if you are still having issues.
